README.md

```diff
@@ -35,6 +35,10 @@ format: winston.format.combine(
 
 Include the necessary `newrelic` CommonJS import prior to the loading of other modules to ensure proper instrumentation.
 
+Note: The stack trace will be written to the error.stack attribute on unhandledException log messages.
```

Comment:
Would use code formatting for `error.stack`, `stack` and `trace` items.

Prob put "New Relic Logs 4000 character..." instead of just "New Relic."

Might be a good place to also mention truncation?

Reply:
Maybe also bold note to stand out?